david_vallejos/space_challenge/src/main/java/com/tech_group/Simulation.java

```diff
@@ -0,0 +1,93 @@
+package main.java.com.tech_group;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+
+/**
+ * Class that contain Simulation methods.
+ */
+public class Simulation {
+
+    /**
+     * Loads items from txt file.
+     * @param fileName path followed with file name.
+     * @return a List of items from txt file.
+     */
+    public ArrayList<Item> loadItems(String fileName) {
+        ArrayList<Item> items = new ArrayList<>();
+
+        try {
+            ArrayList<String> fileList = (ArrayList<String>) Files.readAllLines(Paths.get(fileName), Charset.defaultCharset());
```

Comment:
Here you can avoid the casting using the abstract class when declaring the variable.

Reply:
Done